### DISCUSSION TITLE
FFmpeg license needed for cineast

### DISCUSSION THREAD

**Discussion by newtonsp** [vitrivr/cineast#336] [Q&A] [answered]
Dear developers,

I'm studying the licenses used by vitrivr and associated tools. Then a question arose about FFmpeg. Does anyone know if the LGPL version of FFmpeg is enough to support all cineast requirements? My personal experience doesn't help, as I'm using the standard version of FFmpeg (a GPL one).

If no one knows, I still can uninstall FFmpeg and install the LGPL version to see if it works.

Thank you
Sincerely

**Comment by lucaro** [comment, marked as answer]
Hey,

I'm not sure I completely understand the question. Cineast uses an ffmpeg library wrapper for java that itself is Apache licensed. The ffmpeg library which is dynamically linked through that wrapper is LGPL licensed. It does not use any system-wide ffmpeg instance and relies only on the library. What use-case do you are concerned that this might be an issue?

**Reply by newtonsp** [reply]
Dear Luca,

Your answer is enough, thank you!

I was concerned with an occasional commercial use of a prototype based on cineast. The chance to achieve commercial use is not high, but if it happens, it is important to pay attention to the licenses of all tools and libraries used.

Sincerely.

**Reply by silvanheller** [reply]
We just recently replaced the remaining problematic libraries in https://github.com/vitrivr/cineast/pull/333, so you might want to check that for your prototype.

**Reply by newtonsp** [reply]
Dear Silvan,

Thank you so much. I'll take the changes into account.

Sincerely
